### PR TITLE
Add demo data

### PR DESCRIPTION
**Pull request by AlexFrid**
Adding a section in SurrrealQL for demo data with the first demo dataset - Surreal Deal.

### CODE REVIEW

**Review by naisofly — COMMENTED** [review]
Nice one! Good job Alex!

File: app/templates/docs/surrealql/demo.hbs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	</block>
+</Layout::Blocks>
+
+<Layout::Gap mini />
+
+	<h4>Dowload and import</h4>
+	<p>
+		<Link @link="https://drive.google.com/uc?id=1AlSMBJrncEuqOtbbEXLqtenUb0Z5BjlD&export=download">Click here to download the dataset</Link><br>
+		Once you have <Link @link="docs.cli.start">started the server</Link>, you can use this import command to import the dataset (this command assumes the file is in your downloads folder).<br>
+		The dataset will take a few seconds to import.
```
Comment (Ekwuno) [inline app/templates/docs/surrealql/demo.hbs:43]: The sentences feel a bit disjointed. What do you think about this suggestion @AlexFrid. 
```suggestion
<Link @link="https://drive.google.com/uc?id=1AlSMBJrncEuqOtbbEXLqtenUb0Z5BjlD&export=download"> Download the dataset here</Link>, then <Link @link="docs.cli.start">start the server</Link> and use the <Link  @link="docs.cli.import"> import command </Link> to add the dataset from your "Downloads" folder. Please be aware that the import process might take a few seconds.
```

File: app/templates/docs/surrealql/demo.hbs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	<h3>Surreal Deal - deals so good it's surreal!</h3>
+	<p>
+		<Link @link="https://drive.google.com/uc?id=1AlSMBJrncEuqOtbbEXLqtenUb0Z5BjlD&export=download">Surreal Deal</Link> is a demo dataset based on a simplified e-commerce platform that sells artwork.<br>
+		The dataset is made up of 8 tables using both <Link @link="docs.surrealql.statements.relate">graph relations</Link> and <Link @link="docs.surrealql.datamodel.records">record links</Link>:<br>
+		person, order, product, review, artist, create, avg_product_review, daily_sales.
+	</p>
+	<p>
+		Pink are the <Link @link="docs.surrealql.statements.define.table">standard tables</Link> <br>
+		Purple are the <Link @link="docs.surrealql.statements.relate">edge tables</Link> <br>
+		Gray are the <Link @link="docs.surrealql.statements.define.table">pre-computed table views</Link>
```
Comment (Ekwuno) [inline app/templates/docs/surrealql/demo.hbs:27]:
```suggestion
		In the diagram below, the nodes in pink are the <Link @link="docs.surrealql.statements.define.table">standard tables</Link>, the ones in purple represent the <Link @link="docs.surrealql.statements.relate">edge tables</Link> which shows relationships between records and SurrealDB as a graph database. While the nodes in gray are the <Link @link="docs.surrealql.statements.define.table">pre-computed table views</Link>
```

File: app/templates/docs/surrealql/demo.hbs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	</p>
+</Layout::Text>
+
+<Layout::Gap mini />
+
+<Layout::Text text-l text-f>
+	<h3>Surreal Deal - deals so good it's surreal!</h3>
+	<p>
+		<Link @link="https://drive.google.com/uc?id=1AlSMBJrncEuqOtbbEXLqtenUb0Z5BjlD&export=download">Surreal Deal</Link> is a demo dataset based on a simplified e-commerce platform that sells artwork.<br>
+		The dataset is made up of 8 tables using both <Link @link="docs.surrealql.statements.relate">graph relations</Link> and <Link @link="docs.surrealql.datamodel.records">record links</Link>:<br>
```
Comment (Ekwuno) [inline app/templates/docs/surrealql/demo.hbs:21]: Can we used nested <p> instead of <br/> here?

**Review by Ekwuno — CHANGES_REQUESTED** [review]
@AlexFrid i left a few suggestions, let me know what you think?

File: app/templates/docs/surrealql/demo.hbs (hunk 1 above)
Comment (timpratim) [reply app/templates/docs/surrealql/demo.hbs:43]: +1 @Ekwuno's version.

File: app/templates/docs/surrealql/demo.hbs (hunk 3 above)
Comment (AlexFrid) [reply app/templates/docs/surrealql/demo.hbs:21]: When you nest the paragraphs, it adds a lot of extra space, which I didn't like, but we can add it if you think it looks better?
image

**Pull request merged**